src/ocamlorg_frontend/pages/install.eml

```diff
@@ -137,36 +136,65 @@ Layout.render
+          <li>
+            <h3>Initialise opam</h3>
+
+            <p>
+              Now, we are ready to initialise opam, which will create a default
+              <a href="<%s Url.tutorial "opam-switch-introduction" %>">opam switch</a>.
+              An opam switch is an isolated environment for the OCaml compiler
+              and any packages you install.
+            </p>
+
+            <%s! Copy_to_clipboard.small_code_snippet ~id:"init-opam" "opam init" %>
+
+            <p> During <code>opam init</code>, you will be asked about Unix
+            support infrastructure. You can select option 1 to let opam manage
+            its own instance of Cygwin installation. For more details, see  </p>
 
-              <p>Any problems installing? Be sure to read the
-              <a href="https://gitlab.com/dkml/distributions/dkml/-/releases">latest DkML release notes</a>.
-              And if the release notes aren't relevant to your issue,
-              <a href="https://gitlab.com/dkml/distributions/dkml/-/issues">file a new issue</a>
-              </p>
+          </li>
+
+          <li>
+            <h3>Activate the opam switch</h3>
+            <p>
+              You need to activate the opam switch by running on cmd
+
+              <%s! Copy_to_clipboard.small_code_snippet ~id:"cmd-activate" "for /f\"tokens=*\" %i in ('opam env') do @%i" %>
+
+              or on PowerShell
+
+              <%s! Copy_to_clipboard.small_code_snippet ~id:"powershell-activate" "(& opam env) -split '\r?\n' | ForEach-Object { Invoke-Expression $_ }" %>
+            </p>
           </li>
         </ol>
 
         <p>
-          Congratulations, you have now installed OCaml as well as the <a href="<%s Url.learn_platform %>">OCaml Platform Tools</a>!
+          Congratulations, you have now installed OCaml!</a>!
         </p>
 
-        <p>
-          The <a href="<%s Url.learn_platform %>">OCaml Platform Tools</a> include the <a target="_blank" href="https://dune.build/">build system Dune</a>
-          and provide you with a complete OCaml development environment.
-        </p>
+      </div>
+      <div>
+        <div class="prose dark:prose-invert">
+          <h2>Set Up an OCaml Development Environment</h2>
 
-        <p>Now you are ready to write some OCaml code!</p>
+          <p>
+            The <a href="<%s Url.learn_platform %>">OCaml Platform Tools</a>,
+            which includes the <a target="_blank" href="https://dune.build/">build system Dune</a>,
+            complete your OCaml development environment. To install them in your current opam switch, run this command:
+          </p>
+
+          <%s! Copy_to_clipboard.small_code_snippet ~id:"install-platform-tools" "opam install ocaml-lsp-server odoc ocamlformat utop" %>
+
+          <p>Now you are ready to write some OCaml code!</p>
+        </div>
+        <a class="btn mt-6 gap-2" href="<%s Url.tutorial "tour-of-ocaml" %>">Take A Tour of OCaml <%s! Icons.arrow_small_right "h-6 w-6" %></a>

```

Comment:
As this section is not ready (while very important I reckon), may I still suggest that we "turn into an issue" to introduce it but still merge this PR minus it 'which represent already a strict improvement) immediately?

Reply:
Actually, `opam install ocaml-lsp-server odoc ocamlformat utop` does work out of the box on my windows 11 VM so the whole PR LGTM...